app/controllers/organisations/courses_controller.rb

```diff
@@ -2,16 +2,16 @@ module Organisations
   class CoursesController < ApplicationController
     before_action :authenticate_user!
     before_action :find_organisation_and_course,
-                  only: %i[active_cohorts inactive_cohorts]
+                  only: %i[ongoing_cohorts ended_cohorts]
 
     layout "student"
 
-    def active_cohorts
-      @active_cohorts = find_cohorts(:active)
+    def ongoing_cohorts
```

Comment:
@yash-learner Let's finally stick to terms _active_ and _ended_ and remove the plan of having it as _ongoing_. This will keep the naming consistent across the application, like for example the filters in school admin. 

Please do reflect the change across this PR.